### PR TITLE
fix: defaults.redirect never honored

### PR DESCRIPTION
**Pull request by onlyurei**
Looks like a bug related to operator priority, or logic.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,12 +16,14 @@ function create(options) {
     const _port = redirectPort === 443 ? '' : (': ' + redirectPort)
 
     return function redirectSSL(req, res, next) {
-        const _isHttps = isHTTPS(req, xForwardedProto)
-        const shouldRedirect = _isHttps === false || (redirectUnknown && _isHttps === null) && redirect
-        if (shouldRedirect) {
-            const ـredirectURL = 'https://' + (redirectHost || req.headers.host) + _port + req.url
-            res.writeHead(statusCode, { Location: ـredirectURL })
-            return res.end()
+        if (redirect) {
```
Comment (pi0) [inline index.js]: Thanks for this PR :) What's your idea if we make condition shorter like this: ```
if (!redirect) { 
  return
}
```

Comment (onlyurei) [reply index.js]: Would it be repeating what L29 does?

Comment (pi0) [reply index.js]: Not exactly. Never mind was just a styling concern :)

**Comment by onlyurei** [comment]
Thanks for merging!

**Pull request merged**